test/widgets_test.rb

```diff
@@ -8,6 +8,16 @@ def stub_widget_value(id, value)
 end
 
 describe Users::PasswordWidget do
+  let(:root_user) { Y2Users::User.new("root") }
+  let(:password) { Y2Users::Password.create_plain(pwd) }
+  let(:pwd) { "s3cr3T" }
+
+  before do
+    allow(root_user).to receive(:root?).and_return(true)
+    allow(subject).to receive(:root_user).and_return(root_user)
+    allow(Y2Users::Password).to receive(:create_plain).and_return(password)
```

Comment:
ugh, this looks like overmock to me. You basically kill any creation of password and just always create pwd content? this is very strange for me